vault/token_store.go

```diff
@@ -2190,6 +2204,9 @@ func (ts *TokenStore) handleUpdateRevokeAccessor(ctx context.Context, req *logic
 	if err != nil {
 		return nil, err
 	}
+	if aEntry == nil {
```

Comment:
Worth adding a warning to the response in this case, since we're changing the behaviour?